code/modules/jobs/job_types/security.dm

```diff
@@ -124,7 +124,7 @@ Detective
 
 	outfit = /datum/outfit/job/detective
 
-	access = list(access_sec_doors, access_forensics_lockers, access_morgue, access_maint_tunnels, access_court, access_brig, access_weapons)
+	access = list(access_sec_doors, access_security, access_forensics_lockers, access_morgue, access_maint_tunnels, access_court, access_brig, access_weapons)
```

Comment:
Honestly, if its skeleton crew and there is only a detective in the security force, they should be filling in for the normal officer role until either one joins or the round ends.